<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-6UNA8P7R" failures="1" tests="5" name="Personal Loan" time="0.756" errors="0" timestamp="2023-12-12T18:36:08 EST">
  <testcase classname="org.TestNgProjects.testDemo2" name="fourthTest" time="0.126"/>
  <testcase classname="org.TestNgProjects.testDemo2" name="thirdTest" time="0.001"/>
  <testcase classname="org.TestNgProjects.testDemo1" name="firstTest" time="0.138">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.TestNgProjects.testDemo1.firstTest(testDemo1.java:34)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- firstTest -->
  <testcase classname="org.TestNgProjects.testDemo1" name="secondTest" time="0.003"/>
  <testcase classname="org.TestNgProjects.testDemo1" name="secondTest2" time="0.002"/>
</testsuite> <!-- Personal Loan -->
